src/doc/DocDrawingThread.js

```diff
@@ -156,6 +157,16 @@ class DocDrawingThread extends DrawingThread {
         super.destroy();
     }
 
+    /** @inheritdoc */
+    delete(annotationToRemove: Object, useServer: boolean = true): ?Promise<any> {
```

Comment:
Would this makes sense to add to `annotationThread.delete()` instead of only for drawing threads?

Reply:
I'm assuming that other types already handle this correctly, since this only affects drawing annotations.